global-news-app/src/components/Search/Search.tsx

```diff
@@ -73,6 +73,7 @@ const Search = () => {
           value={option}
           handleChange={setOption}
           options={SearchSelectOptions}
+          fontWeight
```

Comment:
if the value is a boolean - i would recommend to change the name of the prop to be self explanatory, for example isFontWeight

but I think it can be better if fontWeight is optional of type number, and the default is 400, then you can potentially set diffirent values